modules/zivid/image.py

```diff
@@ -0,0 +1,78 @@
+"""Contains a the Image class."""
+import numpy
+
+import _zivid
+
+
+class Image:
+    """A 2-dimensional image stored on the host."""
```

Comment:
What is the host? I see this is just copied from C++, but I have no idea what it means. I would open an issue against C++ SDK if you agree.

Reply:
Will create an issue